component/azstorage/block_blob_test.go

```diff
@@ -3140,6 +3141,85 @@ func (s *blockBlobTestSuite) TestInvalidMD5OnReadNoVaildate() {
 // 	s.az.CloseFile(internal.CloseFileOptions{Handle: h})
 // }
 
+func (suite *blockBlobTestSuite) TestTruncateSmallFileToSmaller() {
+	suite.UtilityFunctionTestTruncateFileToSmaller(20*MB, 10*MB)
+}
+
+func (suite *blockBlobTestSuite) TestTruncateSmallFileToLarger() {
+	suite.UtilityFunctionTruncateFileToLarger(10*MB, 20*MB)
+}
+
+func (suite *blockBlobTestSuite) TestTruncateBlockFileToSmaller() {
+	suite.UtilityFunctionTestTruncateFileToSmaller(300*MB, 290*MB)
+}
+
+func (suite *blockBlobTestSuite) TestTruncateBlockFileToLarger() {
+	suite.UtilityFunctionTruncateFileToLarger(290*MB, 300*MB)
+}
+
+func (suite *blockBlobTestSuite) TestTruncateNoBlockFileToLarger() {
+	suite.UtilityFunctionTruncateFileToLarger(200*MB, 300*MB)
```

Comment:
Why we are not checking any failure or success in these cases, how do we know it works.